src/odin/commands/configure.rs

```diff
@@ -58,6 +58,9 @@ pub struct Configuration {
 
   /// Sets the save interval in seconds
   pub save_interval: Option<u16>,
+
+  /// Steam home dirname
+  pub steam_home_dirname: String,
```

Comment:
the repo is very much coded to only be executing in cointainer. i just _couldnt_ get the container to ever boot. the steam script segfaulting and not emitting any output :( 

 i figured "maybe i can at least unit/integration test my feature" without actually running a real server. however, 'odin configure' and 'odin install' were some pre-reqs that i only _started_ to hack through to test this approach, but well exceeded my timebox today